letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
This issue is not related with dbfilter. If there are two or more DBs after applying dbfilter Odoo can't preload one of them, and HTTP request for ACME validation come with no DB selection, so only ``web`` addon is available for routing map.

I've check in one production environment with this use case, and there are two issues:
1. letsencrypt controller not found. Then we need --load=web,letsencrypt
2. We have no request.env defined, so we can't call to models. Then we can only call to static methods.